### PR TITLE
Allow to configure the error indicator color

### PR DESCRIPTION
**Pull request by b4n**
Related to #1116.

---
trivial enough I guess

### CODE REVIEW

**Comment by b4n** [comment]
I might be interesting to allow a theme to override the default, not sure.

**Comment by b4n** [comment]
@codebrainz @elextr opinions?

**Comment by codebrainz** [comment]
Looks OK at a glance, I haven't tested it though.

**Comment by elextr** [comment]
One document suggestion, otherwise LGBI.

**Comment by elextr** [comment]
PS it probably would be good to make it themable, then a theme suitable for the colour challenged could be created.

File: doc/geany.txt [hunk 1]
```diff
@@ -4593,6 +4593,11 @@ calltips
 
     *Example:* ``calltips=0xc0c0c0;0xffffff;false;false``
 
+indicator_error
+    The color of the error indicator.
+
```
Comment (b4n) [inline doc/geany.txt]: If you want, but as most of these settings use the value differently I thought "the color" was clear enough.  Do you want me to add something like *"Only the first argument (foreground color) is used."*?

**Comment by b4n** [comment]
> PS it probably would be good to make it themable, then a theme suitable for the colour challenged could be created.

Yes.  I wanted to give it a shot, but it was more complex and only ended up with #1186, which still might be fairly useful to get in.

File: doc/geany.txt (hunk 1 above)
Comment (elextr) [inline doc/geany.txt]: Thats fine.

**Pull request merged**